jobs/periodic/drop-old-notification-contexts/index.ts

```diff
@@ -0,0 +1,24 @@
+import {prisma} from "../../../common/prisma";
+import moment from "moment";
+import {getLogger} from "log4js";
+
+const DELETE_AFTER = 30 * 3; // in days
+const logger = getLogger();
+
+export default async function execute() {
+
+    const res = await prisma.concrete_notification.updateMany({
+        where: {
+            sentAt: {
+                lte: moment().startOf("day")
+                    .subtract(DELETE_AFTER, "days")
+                    .toDate()
+            }
+        },
+        data: {
+            context: {}
```

Comment:
If we want to show all notifications to the user in some UI, it might be a good idea to somehow be able to distinguish redacted notifications, maybe we could introduce a new notification state ARCHIVED? In theory an empty notification context could also be valid for a sent notification (though it usually always includes the user)